libnmstate/nm/applier.py

```diff
@@ -132,6 +134,10 @@ def set_ifaces_admin_state(ifaces_desired_state, con_profiles=()):
         nmdev = device.get_device_by_name(ifname)
         if not nmdev:
             if ifname in new_ifaces and iface_desired_state['state'] == 'up':
+                if _is_master_iface(
```

Comment:
Consider for readability (not a must): `if is_master_iface() and not is_slave_iface():`

Reply:
'black' - the code formatter - made me do it! 

Since we do also assert the code format in the tests, this has to stay like this. Right?..

Reply:
Sorry, misinterpreted your comment. Followed your suggestion, and added that method.

Done.